src/protocol/boards/aiy/api.js

```diff
@@ -154,6 +157,27 @@ cwc.protocol.aiy.Api.prototype.sendRun = function(code, args) {
 };
 
 
+/**
+ * @param {!number} timeout
+ * @return {!Promise}
+ * @export
+ */
+cwc.protocol.aiy.Api.prototype.terminateJoyDemo = function(timeout) {
```

Comment:
Could you make the timeout parameter optional but with an default value like (timeout = 2000) so that you only need to provide an timeout if you want something different.

Does it really took 2 secs until the joy demo is stopped? If not please use the smallest possible value for this.

Reply:
Done.

> Does it really took 2 secs until the joy demo is stopped?

Problem is, because of the way things are now, we close the socket which kills the process, so lowering the timeout means it might not stop at all, so I'd rather not risk lowering. This'll go away after I refactor it anyway.